File: examples-sdk/csharp/509-villog-phase1-integration/Main.cs
// 509-villog-phase1-integration — C# SDK equivalent
// Compile: vil compile --from csharp --input 509-villog-phase1-integration/Main.cs --release

#r "sdk/csharp/Vil.cs"

var server = new VilServer("app", 8080);
server.Compile();

File: examples-sdk/zig/509-villog-phase1-integration/main.zig
// 509-villog-phase1-integration — Zig SDK equivalent
// Compile: vil compile --from zig --input 509-villog-phase1-integration/main.zig --release

const vil = @import("vil");

pub fn main() void {
    var server = vil.Server.init("app", 8080);
    server.compile();
}
